testdata/runtests.bash

```diff
@@ -0,0 +1,110 @@
+$exe -c ../test4.cfg ../par1.crs par4.out par4.log
+$exe -c ../test5.cfg ../par1.crs par5a.out par5a.log
+$exe -c ../test5.cfg ../parh.crs par5b.out par5b.log
+$exe -c ../test5.cfg ../parhz.crs.gz par5c.out par5c.log
+$exe -c ../test6.cfg ../par1.crs par6a.out par6a.log
+$exe -c ../test6.cfg ../parh.crs par6b.out par6b.log
+$exe -c ../test1.cfg -p ../scg.crs.p0+../scg.crs.p1.gz ../scg.crs scgp.out scgp.log
+$exe -c ../test1.cfg ../scg.crs+../scg1.crs scg1.out scg1.log
+$exe -c ../test1.cfg -l 0 scg scg2.out scg2.log
+$exe -c ../test1.cfg -l 0 -d 20100401030201 scg scg3.out scg3.log
+$exe -c ../test1.cfg -x -l 0 scg scg4.out scg4.log
+$exe -c ../test1.cfg -x -l 0 -d 20100401030201 scg scg5.out scg5.log
+
+$exe -c ../test1.cfg ../par1.crs -z par6.out.gz par6.log
+
+$exe -c ../test1.cfg -w user_modify_flag=N ../scg.crs scgw1.out scgw1.log
+$exe -c ../test1.cfg -w user_modify_flag=N:data_type!=T ../scg.crs scgw2.out scgw2.log
+
+$exe -c ../testutf1.cfg ../mixutf.crs mixutf1.out testutf1.log
+$exe -c ../testutf2.cfg ../mixutf.crs mixutf2.out testutf2.log
+$exe -c ../testutf3.cfg ../mixutf.crs mixutf3.out testutf3.log
+$exe -c ../testutf4.cfg ../mixutf.crs mixutf4.out testutf4.log
+$exe -c ../testutf5.cfg ../lolutf.crs lolutf.out lolutf.log
+
+# OutputFile as '-' Checks
+$exe -c ../test1.cfg ../par1.crs - par7.log > par7.stdout
+$exe -c ../test2.cfg ../par1.crs - par8.log > par8.stdout
+$exe -c ../test3.cfg ../par1.crs - par9.log > par9.stdout
+$exe -c ../test1.cfg ../escape.crs - escape2.log > escape2.stdout
+$exe -c ../test4.cfg ../par1.crs - par10.log > par10.stdout
+$exe -c ../test5.cfg ../par1.crs - par5a.log
+$exe -c ../test5.cfg ../parh.crs - par11.log > par11.stdout
+$exe -c ../test5.cfg ../parhz.crs.gz - par12.log > par12.stdout
+$exe -c ../test6.cfg ../par1.crs - par13.log > par13.stdout
+$exe -c ../test6.cfg ../par1.crs - par14.log > par14.stdout
+$exe -c ../test1.cfg -p ../scg.crs.p0+../scg.crs.p1.gz ../scg.crs - scgp2.log > scgp2.stdout
+$exe -c ../test1.cfg ../scg.crs+../scg1.crs - scg6.log > scg6.stdout
+$exe -c ../test1.cfg -l 0 scg - scg7.log > scg7.stdout
+$exe -c ../test1.cfg -x -l 0 -d 20100401030201 scg - scg8.log > scg8.stdout
+
+$exe -c ../testutf1.cfg ../mixutf.crs - testutf5.log > mixutf5.stdout
+$exe -c ../testutf5.cfg ../lolutf.crs - lolutf1.log > lolutf1.stdout
+$exe -c ../test1.cfg -w user_modify_flag=N ../scg.crs - scgw3.log > scgw3.stdout
+
+# Test diff paramater options.
+$exe -c ../test1.cfg -o id:area:shape ../par1.crs - par1o.log > par1o.stdout
+$exe -c ../test1.cfg -a ../par1.crs - par1a.log > par1a.stdout
+$exe -c ../test1.cfg -n ../par1.crs - par1n.log > par1n.stdout
+$exe -c ../test1.cfg -h ../par1.crs - par1h.log
+
+# Redirects stdout to file, and redirects stderr through stdout to pipt the first line of stderr to file.
+# Removes the stdout file if empty.
+$exe -c ../test1.cfg ../par1.crs -+ par1f1.out 2>&1 >par1f1.stdout | head -n1 > par1f1.stderr
+if [ ! -s par1f1.stdout ]
+then rm par1f1.stdout
```

Comment:
It would be good to put the `rm` command indented on the next line - this formatting is very unusual.